### DISCUSSION TITLE
Why are relations getting optional typing?

### DISCUSSION THREAD

**Discussion by axhamre** [prisma/prisma#15937] [Q&A]
Presume we have a normal one-to-many relation as the example below ([taken from the docs](https://www.prisma.io/docs/concepts/components/prisma-schema/relations)).

```
model User {
  id    Int    @id @default(autoincrement())
  posts Post[]
}

model Post {
  id       Int  @id @default(autoincrement())
  author   User @relation(fields: [authorId], references: [id])
  authorId Int 
}
```

When creating a Post, the generated types will have `authorId` as optional. Why is that? And is there any way to prevent the optionality?

**Comment by austincrim** [comment]
Hey @axhamre 👋,

I'm not seeing the situation you describe. With the above schema, I get this type definition:
![CleanShot 2022-10-24 at 14 21 04](https://user-images.githubusercontent.com/32459922/197608264-053c14f7-7254-4b59-a520-d5753546af3e.png)

Could you describe precise reproduction steps?
What versions of Prisma are you on?

**Reply by anitacung** [reply]
👋 Hi @axhamre,

I'm going through pending GitHub Discussions to check if you need any help. I see that Austin asked a few clarification questions, have you looked into this yet? 

We want to be sure that all our users are successful and our doors are always open if you ever want to share some feedback with our Product team. 🤗

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
